guides/common/modules/proc_creating-hosts-with-pxeless-provisioning.adoc

```diff
@@ -68,73 +80,87 @@ Note in particular:
-endif::[]
-ifdef::satellite,orcharhino[]
+ifdef::katello,satellite,orcharhino[]
 . Click the *Parameters* tab, and ensure that a parameter exists that provides an activation key.
 If not, add an activation key.
 endif::[]
 . Click *Submit* to save the host details.
-
 This creates a host entry and the host details page appears.
+. Download the boot disk from {ProjectServer}.
++
+ifndef::satellite[]
+* For *Host image*, on the host details page, click the vertical elipsis and select *Host '_My_Host_Name_' image*.
+endif::[]
+* For *Full host image*, on the host details page, click the vertical elipsis and select *Full host '_My_Host_Name_' image*.
+ifndef::satellite[]
+* For *Generic image*, navigate to *Infrastructure* > *Subnets*, click *Boot disk* and select *Generic image*.
+endif::[]
+* For *Subnet image*, navigate to *Infrastructure* > *Subnets*, click the dropdown menu in the *Actions* column of the required subnet and select *Subnet generic image*.
+. Write the ISO to a USB storage device using the *dd* utility or *livecd-tools* if required.
+. When you start the host and boot from the ISO or the USB storage device, the host connects to {ProjectServer} and starts installing operating system from its kickstart tree.
+ifdef::satellite,katello,orcharhino[]
++
+When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
+endif::[]
+
 
-The options on the upper-right of the window are the *Boot disk* menu.
-From this menu, one of the following images is available for download: *Host image*, *Full host image*, *Generic image*, and *Subnet image*.
 
 [id="cli-creating-hosts-with-pxe-less-provisioning_{context}"]
 .CLI procedure
 . Create the host with the `hammer host create` command.
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer host create --name "_My_Bare_Metal_" --organization "_My_Organization_" \
---location "_My_Location_" --hostgroup "_My_Host_Group_" --mac "aa:aa:aa:aa:aa:aa" \
+# hammer host create --name "_My_Host_Name_" --organization "_My_Organization_" \
+--location "_My_Location_" --hostgroup "_My_Host_Group_" --mac "_aa:aa:aa:aa:aa:aa_" \
 --build true --enabled true --managed true
 ----
 . Ensure that your network interface options are set using the `hammer host interface update` command.
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer host interface update --host "test3" --managed true \
+# hammer host interface update --host "_My_Host_Name_" --managed true \
 --primary true --provision true
 ----
-. Download the boot disk from {ProjectServer} with the `hammer bootdisk host` command:
+. Download the boot disk from {ProjectServer} with the `hammer bootdisk` command:
 +
+ifndef::satellite[]
 * For *Host image*:
 +
 [options="nowrap" subs="+quotes"]
 ----
-# hammer bootdisk host --host _test3.example.com_
+# hammer bootdisk host --host _My_Host_Name.example.com_
```

Comment:
This is OK for me for now, but it reinforces my though I had this week to introduce an attribute like `client-example-com`. I'll do this eventually in a separate PR.

Reply:
If you want to take the initiative, let's do this in another PR, because it might be needed in other modules as well and that's out of scope of this PR.

Reply:
Yes, works for me!